README.md

```diff
@@ -1,155 +1,49 @@
-7. The plugin is now ready to use.
-
-### Configure the configuration files watcher
-
-*This step is optional, although highly recommended.*
-
-Regenerate the token for the configuration files watcher:
-1. In Mattermost, go to **System Console > Plugins > Nagios**.
-2. Select **Regenerate** to regenerate the token.
-3. Copy the token - you're going to use it later.
-4. Select **Save**.
-5. Switch to the machine where Nagios is running:
-  - Download the latest stable version of the watcher from the [releases page](https://github.com/mattermost/mattermost-plugin-nagios/releases)
-  - Move the watcher: `chmod +x watcher1.1.0.linux-amd64 && sudo mv watcher1.1.0.linux-amd64 /usr/local/bin/watcher`.
-
-### Run the watcher as a systemd service
-
-#### Prepare the systemd service unit file
-
-Prepare the systemd service unit file by adjusting `dir` (default if not set: `/usr/local/nagios/etc/`), `url`, and `token` flags to your setup.
-
-```sh
-sudo bash -c 'cat << EOF > /etc/systemd/system/mattermost-plugin-nagios-watcher.service
-[Unit]
-Description=Nagios configuration files monitoring service
-After=network.target
-
-[Service]
-Restart=on-failure
-ExecStart=/usr/local/bin/watcher -dir /nagios/configuration/files/directory -url https://mattermost.server.address/plugins/nagios -token TheTokenFromStep1
-
-[Install]
-WantedBy=multi-user.target
-EOF'
-```
-
-#### Start the watcher
-
-```sh
-systemctl daemon-reload
-systemctl enable mattermost-plugin-nagios-watcher.service
-systemctl start  mattermost-plugin-nagios-watcher.service
-```
-
-### Get logs
-![logs](https://github.com/mattermost/mattermost-plugin-nagios/assets/74422101/7701b754-dc79-4199-9779-a52c173fbbba)
-
-### Receive system monitoring reports
-![reports](https://github.com/mattermost/mattermost-plugin-nagios/assets/74422101/c4369e89-a518-498a-841e-a608dacce40c)
-
-### Receive notifications about changes to the configuration
-![changes](https://github.com/mattermost/mattermost-plugin-nagios/assets/74422101/d8920978-c184-45bd-b443-cdae31a979e4)
+[![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-nagios/master)](https://circleci.com/gh/mattermost/mattermost-plugin-nagios)
+[![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-nagios/master)](https://codecov.io/gh/mattermost/mattermost-plugin-nagios)
+[![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-nagios)](https://github.com/mattermost/mattermost-plugin-nagios/releases/latest)
+[![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-nagios/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-nagios/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-Ultimately, this will make you or your team more productive and make the experience with Nagios smoother.
+**Help Wanted Tickets [here]**
```

Comment:
Is there a link missing?

Reply:
There is yes: ``https://github.com/mattermost/mattermost-plugin-nagios/issues``

Reply:
Nice catch, @hanzei!